ui/pages/snaps/snap-view/snap-settings.js

```diff
@@ -146,8 +152,37 @@ function SnapSettings({ snapId }) {
     setIsDescriptionOpen(true);
   };
 
+  const snapLatestVersion = useSelector((state) =>
+    snap ? getSnapLatestVersion(state, snap?.id) : null,
+  );
+  const checkAvailableUpdate = () => {
```

Comment:
Do we need a function here? Could we not just do:
```
const isUpdateAvailable = snapLatestVersion
      ? semver.gt(snapLatestVersion, snap.version)
      : false
```

Reply:
Updated / refactored.